src/pages/api/repositories/taskRepository.ts

```diff
@@ -0,0 +1,10 @@
+import { TaskModel } from '../database/cassandra';
+import { BaseRepository } from './baseRepository';
+
+class TaskRepository extends BaseRepository<typeof TaskModel> {
```

Comment:
Move this file away from api folder. Everything inside that can be called by a client.